sdk/storage/Azure.Storage.Blobs.Batch/samples/Azure.Storage.Blobs.Batch.Samples.Tests.csproj

```diff
@@ -15,6 +15,9 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="C:\src\azure-sdk-for-net\sdk\storage\Azure.Storage.Blobs.Batch\samples\..\..\Azure.Storage.Common\tests\Shared\StorageRecordMatcher.cs" />
```

Comment:
hm

Reply:
StorageRecordMatcher.cs was deleted.

Reply:
Sure, so why does this file have a local path in it?

Reply:
oh - this is something strange that VS did. I was reading this as being deleted for some reason, but it added this :/